### PR TITLE
Make medicine prices tool embeddable

### PR DESCRIPTION
**Pull request by RyanSept**
### Description of Task to be completed?

Make the medicine prices tool embeddable and a couple of stylistic changes.

### How should this be manually tested?

Open medprices.codefornigeria.org site and click embed this on the navbar to get the embed code which you can place on a site or navigate to medprices.codefornigeria.org/#/embed.

### CODE REVIEW

File: modules/home.html [hunk 1]
```diff
@@ -1,138 +1,84 @@
...
-
-		  ga('create', 'UA-89116337-1', 'auto');
+			(function (i, s, o, g, r, a, m) {
+			i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
+				(i[r].q = i[r].q || []).push(arguments)
+			}, i[r].l = 1 * new Date(); a = s.createElement(o),
+				m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
+			})(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
 
+			ga('create', 'UA-89116337-1', 'auto');
```
Comment (DavidLemayian) [inline modules/home.html]: Change Google Analytics tracking code to `UA-93090833-2`.

File: modules/home.html [hunk 2]
```diff
@@ -1,138 +1,84 @@
...
+			$('.dropdown-menu').click(function(e) {
+				e.stopPropagation(); //keeps dropdown open
+			});
 		</script>
 		<div class="row">
 			<div class="col-md-12 tool-bar no-padder">
 				<a href="http://www.codefornigeria.org"><img src="assets/img/logo.png" class="logo"></a>
-				<a href="https://twitter.com/codefornigeria" target="_blank"><li class="fa fa-twitter"></li></a>
-				<a href="https://web.facebook.com/codefornigeria" target="_blank"><li class="fa fa-facebook"></li></a>
+				<a href="https://twitter.com/codefornigeria" target="_blank">
```
Comment (DavidLemayian) [inline modules/home.html]: Change twitter URL to `https://twitter.com/code4africa`.

File: modules/home.html [hunk 3]
```diff
@@ -1,138 +1,84 @@
...
+			})(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
 
+			ga('create', 'UA-89116337-1', 'auto');
+			$('.dropdown-menu').click(function(e) {
+				e.stopPropagation(); //keeps dropdown open
+			});
 		</script>
 		<div class="row">
 			<div class="col-md-12 tool-bar no-padder">
 				<a href="http://www.codefornigeria.org"><img src="assets/img/logo.png" class="logo"></a>
```
Comment (DavidLemayian) [inline modules/home.html]: Logo in navbar should link to `/`, index page.

File: modules/home.html [hunk 4]
```diff
@@ -1,138 +1,84 @@
...
-				<a href="https://twitter.com/codefornigeria" target="_blank"><li class="fa fa-twitter"></li></a>
-				<a href="https://web.facebook.com/codefornigeria" target="_blank"><li class="fa fa-facebook"></li></a>
+				<a href="https://twitter.com/codefornigeria" target="_blank">
+					<li class="fa fa-twitter"></li>
+				</a>
+				<a href="https://web.facebook.com/codefornigeria" target="_blank">
```
Comment (DavidLemayian) [inline modules/home.html]: Let's change Facebook to point to `https://facebook.com/CodeForAfrica/`

File: index.html [hunk 5]
```diff
@@ -53,7 +54,10 @@
     <script src="assets/vendor/angular/angulartics/dist/angulartics.min.js"></script>
     <script src="assets/vendor/angular/angulartics/dist/angulartics-gtm.min.js"></script>
     <script src="assets/vendor/angular/angulartics-google-analytics/dist/angulartics-ga.min.js"></script>
+    <script src="https://cdn.rawgit.com/zenorocha/clipboard.js/master/dist/clipboard.min.js"></script>
+    <script src="assets/vendor/angular/ng-clipboard/ngclipboard.min.js"></script>
```
Comment (RyanSept) [inline index.html]: Hi according to the documentation both are need and on checking the content they seem to be different files. I reckon the ngclipboard is an angular wrapper for the clipboard js .

File: modules/home.html [hunk 6]
```diff
@@ -70,12 +73,12 @@
...
 		<div class="col-md-12 partners">
-			<p class="footer-header">Built by Code for Nigeria, with thought leadership from</p>
+			<p class="footer-header">Built by <a href="https://codefornigeria.org/">Code for Nigeria</a>, with thought leadership from</p>
 			<hr>
 			<a href="http://www.icfj.org" target="_blank"><img src="assets/img/icfj.png"></a>
 			<a href="http://www.codeforafrica.org" target="_blank"><img src="assets/img/cfafrica.png"></a>
 		</div>
 
-		<p class="text-center text-sm text-thin"><a href="https://github.com/Code4Nigeria/medicine-prices-app" target="_blank">Fork this project on Github</a></p>
+		<p class="text-center text-sm text-thin">&nbsp</p>
```
Comment (DavidLemayian) [inline modules/home.html]: @RyanSept Let's remove this unused paragraph.

File: modules/home.html [hunk 7]
```diff
@@ -45,21 +48,21 @@
...
 <div class="container-fluid main-content-container">
-
 	<main-content></main-content>
 
 	<div class="footer">
 		<div class="col-md-12 no-padder min150 m-b-md">
 			<div class="col-md-8 min150">
 				<p class="footer-header">About</p>
-				<hr> Code for Nigeria is a non-profit civic technology movement. A network of civic-minded volunteers who contribute their
+				<hr> <a href="https://codefornigeria.org/">Code for Nigeria</a> is a non-profit civic technology movement. A network of civic-minded volunteers who contribute their
```
Comment (DavidLemayian) [inline modules/home.html]: Let this open in a new tab.

**Pull request merged**